GitExtensions.sln

```diff
@@ -354,8 +354,8 @@ Global
 		{2B9B62FA-3E0C-45C5-9287-68B69413C5DF}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{2B9B62FA-3E0C-45C5-9287-68B69413C5DF}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{2B9B62FA-3E0C-45C5-9287-68B69413C5DF}.Release|Any CPU.Build.0 = Release|Any CPU
-		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.ActiveCfg = Release|Any CPU
-		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.Build.0 = Release|Any CPU
+		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.Build.0 = Debug|Any CPU
```

Comment:
could you please clarify the nature of this change?

Reply:
Before debugging Visual Studio blames about trying to debug Release configuration ICSharpCode.TextEditor

the same as that:
![image](https://user-images.githubusercontent.com/12046452/44977674-4ba3ee80-af71-11e8-8ea6-87d77dee18f9.png)

As this change is unrelated to pale selection, it's in a separate commit `fix solution configuration`

Reply:
PR looks good, though please revert the change to the configuration.

Reply:
ok